### ISSUE TITLE
Reference git Repository

### ISSUE THREAD

**Issue by jooyounghun** [jooyounghun/AI-Team-5#1]
### Mask-RCNN

https://github.com/jooyounghun/Mask_RCNN

Tutorial Interpreter.
jupyter version

/sample -> demo.ipynb 를 돌려보면서 전체적인 흐름을 읽으면 좋을 것 같습니다.

**Comment by MH-Lee** [comment]
https://github.com/chenyuntc/simple-faster-rcnn-pytorch/blob/master/demo.ipynb

**Comment by MH-Lee** [comment]
https://github.com/inspace4u/keras-frcnn

**Comment by MH-Lee** [comment]
https://github.com/chenyuntc/simple-faster-rcnn-pytorch/blob/master/demo.ipynb

https://github.com/jinfagang/keras_frcnn

https://github.com/DetectionTeamUCAS/Faster-RCNN_Tensorflow

조금 간단해 보이는 repository를 몇개 찾았습니다.
